osu.Game/Overlays/SocialOverlay.cs

```diff
@@ -34,6 +34,8 @@ public class SocialOverlay : SearchableListOverlay<SocialTab, SocialSortCriteria
         protected override SearchableListHeader<SocialTab> CreateHeader() => new Header();
         protected override SearchableListFilterControl<SocialSortCriteria, SortDirection> CreateFilterControl() => new FilterControl();
 
+        protected override string LoginPlaceholder => @"Please sign in to browse social!";
```

Comment:
this sounds really weird.

just make it "Please sign in to view content" for all cases for now.